x-pack/legacy/plugins/code/kibana.json

```diff
@@ -2,7 +2,7 @@
   "id": "code",
   "version": "kibana",
   "server": true,
-  "ui": true,
+  "ui": false,
```

Comment:
should I disable the `ui` in the legacy plugin settings?

Reply:
`kibana.json` is handled for new platform plugins only. This file is ignored because resides in `legacy` folder.